### ISSUE TITLE
Bug: modal makes itself aria-hidden

### ISSUE THREAD

**Issue by helixbass** [react-bootstrap/react-overlays#292]
When using a `react-bootstrap` `Modal`, I'm unable to navigate the modal using a screen reader

Inspecting the DOM, I see that the outer `<div role="dialog" aria-modal="true">` element has `aria-hidden="true"` set on it (while the modal is open)

I tracked down the logic to what looks like a simple bug fix in `src/utils/manageAriaHidden.js`. Will submit a fixing PR momentarily
